src/qudi/hardware/ni_x_series/ni_x_series_finite_sampling_input.py

```diff
@@ -60,6 +60,7 @@ class NIXSeriesFiniteSamplingInput(FiniteSamplingInputInterface):
             max_channel_samples_buffer: 10000000  # optional, default 10000000
             read_write_timeout: 10  # optional, default 10
             sample_clock_output: '/Dev1/PFI20'  # optional
+            invert_trigger_polarity: False  # optional
```

Comment:
Maybe the naming here might be a bit confusing, because it is tied to a specific hardware which is connected in the end.
We could use this opportunity of your PR and change this to an explicit config option, like "`_trigger_polarity: RISING`"?
There is a nifty feature in `qudi-core` for config options: a [`constructor`](https://github.com/Ulm-IQO/qudi-core/blob/8ace055a0aff38ee9e166816b12712c0337038ef/src/qudi/core/configoption.py#L1109)